src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({
+  tabs,
+  selectedTab,
+  onTabSelected,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={selectedTab.id === tab.id ? 'is-active' : ''}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (selectedTab.id !== tab.id) {
+                  onTabSelected(tab);
+                }
+              }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here